dts/arm/st/u5/stm32u5.dtsi

```diff
@@ -662,6 +664,20 @@
 			dma-offset = <0>;
 			status = "disabled";
 		};
+
+		fmc: memory-controller@420d0400 {
+			compatible = "st,stm32-fmc";
+			reg = <0x420d0400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00001000>;
```

Comment:
can you please confirm, I see the FSMC clock control on the  AHB2_2 bit 0   (AHB3 bit 12 is GTZC)

Reply:
You are right, I double checked in the documentation. I have to say, I just took over the configuration from H7, which did not align with U5. I fixed the clocks definition.

Reply:
One question then: did you test it ?

Reply:
Not yet, as the hardware is not yet ready. Will be able to test it next week.

Reply:
ok. Then I prefer we don't merge and wait for tests results.
It's possible that FMC on U5 requires more than only dtsi addition. Some drivers changes might be required as well. This is usual on U5 series.

Reply:
@potentialdiffer please add test configuration as part of this PR in a dedicated commit